h3.go

```diff
@@ -246,8 +248,33 @@ func (p GeoPolygon) Cells(resolution int) []Cell {
 	return PolygonToCells(p, resolution)
 }
 
+func CellToGeoPolygon(cell Cell) GeoPolygon {
+	boundary := CellToBoundary(cell)
+	loop := make(GeoLoop, len(boundary))
+
+	for i, coord := range boundary {
+		loop[i] = LatLng{Lat: coord.Lat, Lng: coord.Lng}
+	}
+
+	return GeoPolygon{GeoLoop: loop}
+}
+
 func CellsToMultiPolygon(cells []Cell) *LinkedGeoPolygon {
-	panic("not implemented")
+	var head, current *LinkedGeoPolygon
```

Comment:
I'd expect this function to wrap `cellsToMultiPolygon` from the C core library, which handles details like ensuring duplicated internal vertices are not included. https://github.com/uber/h3/blob/138a7487adb0892715ea7ced785db4e9c38842d1/src/h3lib/include/h3api.h.in#L293